migrations/versions/840_add_application_close_date_for_framework.py

```diff
@@ -0,0 +1,37 @@
+"""Add `application_close_date` for Framework
+Revision ID: 840
+Revises: 830
+Create Date: 2017-02-06 11:09:26.852142
+"""
+from alembic import op
+from datetime import datetime
+import sqlalchemy as sa
+
+from dmutils.formats import DATETIME_FORMAT, EUROPE_LONDON
+
+# revision identifiers, used by Alembic.
+revision = '840'
+down_revision = '830'
+
+
+
+def upgrade():
+    op.add_column('frameworks', sa.Column('application_close_date', sa.DateTime(), nullable=True))
+    op.add_column('frameworks', sa.Column('allow_declaration_reuse', sa.Boolean(), nullable=False, server_default='false'))
+
+    for framework_slug, a_c_date, declaration_reuse in (
+        ('digital-outcomes-and-specialists', datetime(2016, 1, 1, 15, tzinfo=EUROPE_LONDON), False),
+        ('digital-outcomes-and-specialists-2', datetime(2017, 1, 16, 17, tzinfo=EUROPE_LONDON), True),
+        ('g-cloud-8', datetime(2016, 6, 1, 17, tzinfo=EUROPE_LONDON), True),
+
+    ):
+        query_string = """UPDATE frameworks set application_close_date = '{a_c_date}'::DATE where slug = '{framework_slug}'"""
+        op.execute(query_string.format(framework_slug=framework_slug, a_c_date=a_c_date.strftime(DATETIME_FORMAT)))
```

Comment:
@risicle 
Explicitly no parameter handling:
https://github.com/ideadevice/alembic/blob/master/alembic/operations.py#L831

I side with the 'Parting Words' section of the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html ) view when it comes to why 'local' code style is important
(See migrations 430, 610, 160 and 340)

Also also:
> really bad form 
> very bad habit
are erring on the negative side of ways to address a colleagues contribution. I've found I catch more flies with honey with this sort of thing. I do appreciate you looking at PRs though, thanks :100: 

Changed the dates, it doesn't make a lot of odds to the end user, I don't mind either way, one less import.
Changed the migration. Not very in line with the codebase but does avoid strings.

@Wynndow 
As a standard for this sort of ORM data migration you'd generally either import the model (less good) or craft your own model (gooder) with the required fields. That way you never have to write any SQL (easy to mess up), you ensure that the ORM knows what's going on with the state of the db (because you're using its model) _and_ you remain consistent with the rest of your codebase with regards to accessing/ updating db records (easy for other people to read).
This Github answer explains the 'how' quite well:
http://stackoverflow.com/a/24623979

@here I crafted my own to see how it worked out :)